service/src/test/java/uk/gov/hmcts/reform/fpl/service/OrganisationServiceTest.java

```diff
@@ -48,6 +51,8 @@
     private static final String AUTH_TOKEN_ID = "Bearer authorisedBearer";
     private static final String SERVICE_AUTH_TOKEN_ID = "Bearer authorised service";
     private static final String USER_EMAIL = "test@test.com";
+    private static final Organisation EMPTY_ORGANISATION = Organisation.builder().build();
```

Comment:
there are other places you do this, I like this private static final declaration, the most. Let's implement it the same in other tests please :)

Reply:
done :D